cafe_information

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE CAFES (
+	name VARCHAR(128) NOT NULL,
+	address VARCHAR(512) NOT NULL,
+	latitude DOUBLE(18,14) NOT NULL,
+	longitude DOUBLE(18,14) NOT NULL,
+	rating FLOAT NOT NULL,
+	monday_open_hour CHAR(11) NOT NULL,
+	tuesday_open_hour CHAR(11) NOT NULL,
+	wednesday_open_hour CHAR(11) NOT NULL,
+	thursday_open_hour CHAR(11) NOT NULL,
+	friday_open_hour CHAR(11) NOT NULL,
+	saturday_open_hour CHAR(11) NOT NULL,
+	sunday_open_hour CHAR(11) NOT NULL,
+	photourl TEXT(512) NOT NULL,
+	take_out_available BIT(1) NOT NULL,
+	parking_available BIT(1) NOT NULL,
+	bike_parking_available BIT(1) NOT NULL,
+	good_for_groups BIT(1) NOT NULL,
+	ambience VARCHAR(64) NOT NULL,
+	wi_fi_available BIT(1) NOT NULL
```

Comment:
we can either do wifi or wi-fi. Personally, 'wifi_available' looks more simple :)